src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionCoordinatorModelTests.swift

```diff
@@ -612,7 +612,52 @@ class ExposureSubmissionCoordinatorModelTests: CWATestCase {
 		waitForExpectations(timeout: .short)
 		XCTAssertEqual(isLoadingValues, expectedIsLoadingValues)
 	}
+	
+	func testGetFamilyMemberTestResultForPCRTestSucceeds() {
+		let expectedFamilyMemberCoronaTest: FamilyMemberCoronaTest = .pcr(.mock(testResult: .positive))
 
+		let coronaTestService = MockCoronaTestService()
+		let familyMemberCoronaTestService = MockFamilyMemberCoronaTestService()
+		familyMemberCoronaTestService.registerPCRTestAndGetResultResult = .success(expectedFamilyMemberCoronaTest)
+
+		let model = ExposureSubmissionCoordinatorModel(
+			exposureSubmissionService: MockExposureSubmissionService(),
+			coronaTestService: coronaTestService,
+			familyMemberCoronaTestService: familyMemberCoronaTestService,
+			eventProvider: MockEventStore()
+		)
+
+		let expectedIsLoadingValues = [true, false]
+		var isLoadingValues = [Bool]()
+
+		let isLoadingExpectation = expectation(description: "isLoading is called twice")
+		isLoadingExpectation.expectedFulfillmentCount = 2
+
+		let onSuccessExpectation = expectation(description: "onSuccess is called")
+
+		let onErrorExpectation = expectation(description: "onError is not called")
+		onErrorExpectation.isInverted = true
```

Comment:
Could this be avoided? Because it will cause the test to definitely wait for this expectation.
Would be calling XCTFail(...) in the onError be sufficient enough?

Reply:
Fixed